ui/style.css

```
@@ -1729,6 +1729,20 @@ span.tag-input[contenteditable]:empty::before {
     animation: spin 2s linear infinite;
 }
 
+.leaderboard-loader {
+    margin-top: 5vh;
+    margin-left: auto;
+    margin-right: auto;

```

This can actually be refactored to:
```suggestion
    margin: 5vh auto 0 auto;
```

---

Did that thank you <3